src/main/java/me/totalfreedom/totalfreedommod/command/Command_wildcard.java

```diff
@@ -38,7 +47,7 @@ public boolean run(CommandSender sender, Player playerSender, Command cmd, Strin
 
         if (BLOCKED_COMMANDS.contains(runCmd.getName()))
         {
-            msg("Did you really think that was going to work?", ChatColor.RED);
+            msg("Did you really think that was going to work? What where you thinking?", ChatColor.RED);
```

Comment:
Probably would change this message all together myself. More along the lines of "Sorry that command cannot be used with the Wildcard command. Please try something else"